scripts/build-kustomize.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+set -eEuo pipefail
+
+git_root_directory="$(git rev-parse --show-toplevel)"
+kustomize_version=$(grep 'sigs.k8s.io/kustomize/kustomize/v5' $git_root_directory/go.mod | awk '{print $2}')
```

Comment:
No need to do `grep ... | awk ...` - awk can perform grep natively.
```suggestion
kustomize_version=$(awk '/sigs\.k8s\.io\/kustomize\/kustomize\/v5/ {print $2}' $git_root_directory/go.mod)
```